src/components/Screen.js

```diff
@@ -1,10 +1,23 @@
 import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
 import './Screen.css';
 
 class Screen extends PureComponent {
   render() {
-    return <input className="screen" value="0" readOnly />;
+    const { total, next, operation } = this.props;
+    return (
+      <div className="screen">
+        <p>{next || total}</p>
+        <p className="operation">{operation || ''}</p>
+      </div>
+    );
   }
 }
 
+Screen.propTypes = {
+  next: PropTypes.number.isRequired,
+  total: PropTypes.number.isRequired,
+  operation: PropTypes.bool.isRequired,
+};
+
```

Comment:
- [ ] [OPTIONAL] This is marked optional because the previous reviewer missed it.
There are errors in the browser console as a result of your specifying a prop to be required while its value is null. Kindly see the errors below and fix it 🙏 
![image](https://user-images.githubusercontent.com/46839548/183049909-a2210b4b-1c4f-447c-9b0d-7fb3a3989a7e.png)

Reply:
Thank you, I will fix it.